### DISCUSSION TITLE
Documentation Feedback

### DISCUSSION THREAD

**Discussion by lantos1618** [oscartbeaumont/rspc#205] [General]
Hey great work on the project so far.

I would like to provide some constructive feedback.

Could you please make sure you qualify the imports on the documentation? This way when potential new users land on the site they know what is being imported from what.

Also clarifying some new user journeys /stories could provide some helpful feedback on documentation to the site

Example.
<img width="888" alt="image" src="https://github.com/oscartbeaumont/rspc/assets/39604275/8d15f488-4b50-42c5-a59a-eb19d062f9f5">

**Comment by oscartbeaumont** [comment]
We are currently working on a rewrite of the docs so hopefully we can do this.
